src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -46,28 +46,20 @@ internal class RoslynPackage : AbstractPackage
         {
             await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
 
+            await InitializeFromBackgroundAsync(cancellationToken).ConfigureAwait(true);
```

Comment:
This method name and placement seems to assume that the calling `InitializeAsync` method is on a background thread. But that's not guaranteed by the `AsyncPackage` base class. Rather, it might be called on any method. So if you really want this on a background thread, you need to switch explicitly. And as our recommendation is for async methods to switch themselves to the thread they require, `InitializeFromBackgroundAsync` could do this internally to itself, in which case its method name can be simplified to stop calling out its thread requirement.